debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/SourceInfo.java

```diff
@@ -75,11 +75,13 @@
     public static final String TXID_KEY = "txId";
     public static final String XMIN_KEY = "xmin";
     public static final String LSN_KEY = "lsn";
+    public static final String SEQUENCE_KEY = "sequence";
```

Comment:
I am not sure about the name. I'd rather have something like `seqNum`, ` orderNum` etc.
I know it was named as such in the Jira comment but I am unsure if it is logically/grammtically correct.
@gunnarmorling WDYT?

Reply:
I left the name as is, happy to alter it to whatever is suggested!

Reply:
I think "sequence" is ok; in particular other connectors may use string attributes as part of that, so I wouldn't add an expection around all parts being numbers to it.